TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/ProcessEndedEmploymentsJob.cs

```diff
@@ -0,0 +1,12 @@
+using TeachingRecordSystem.Core.Jobs.Scheduling;
+using TeachingRecordSystem.Core.Services.WorkforceData;
+
+namespace TeachingRecordSystem.Core.Jobs;
+
+public class ProcessEndedEmploymentsJob(IBackgroundJobScheduler backgroundJobScheduler)
+{
+    public async Task ExecuteAsync(CancellationToken cancellationToken)
+    {
+        await backgroundJobScheduler.Enqueue<TpsCsvExtractProcessor>(j => j.ProcessEndedEmployments(cancellationToken));
```

Comment:
Is there any need to push this into its own job? Could we just process here instead?